packages/psss/src/api/queries/useSingleWeeklyReport.js

```diff
@@ -55,15 +57,15 @@ const getAlerts = row =>
     [],
   );
 
-export const useSingleWeeklyReport = (orgUnit, period, verifiedStatuses, pageQueryKey) => {
-  const query = useData(
-    `weeklyReport/${orgUnit}`,
+const useCachedQuery = (endpoint, orgUnit, period, queryKey) => {
```

Comment:
Great work extracting functionality in a new method! We may even find out that this can be reused in other queries - but till then, we can keep it local to this file